src/niweb/apps/noclook/templatetags/noclook_tags.py

```diff
@@ -24,6 +24,34 @@ def type_menu():
     return {'types': types}
 
 
+def mode_menu():
+    """
+    Adds a the menu items if it's set in the dynamic_preferences
+    """
+    global_preferences = global_preferences_registry.manager()
+    menu_mode = global_preferences['general__menu_mode']
+
+    if menu_mode == 'ni':
```

Comment:
If you return the actual value instead of a bool this can be extended in an easy way with something like,

`{% ifequal val "ni" %}
{% endifequal %}`